packages/react-scripts/backpack-addons/splitChunks.js

```diff
@@ -4,23 +4,31 @@ const paths = require('../config/paths');
 const appPackageJson = require(paths.appPackageJson);
 const bpkReactScriptsConfig = appPackageJson['backpack-react-scripts'] || {};
 
+const chunksAndGroups = {
+  chunks: 'all',
+  cacheGroups: bpkReactScriptsConfig.vendorsChunkRegex
+    ? {
+        defaultVendors: {
+          test: new RegExp(bpkReactScriptsConfig.vendorsChunkRegex),
+        },
+      }
+    : {},
+};
+
 module.exports = (isEnvDevelopment) => {
   // Automatically split vendor and commons
   // https://twitter.com/wSokra/status/969633336732905474
   // https://medium.com/webpack/webpack-4-code-splitting-chunk-graph-and-the-splitchunks-optimization-be739a861366
   return {
     splitChunks: bpkReactScriptsConfig.enableAutomaticChunking
-    ? {
-      chunks: 'all',
-      name: isEnvDevelopment,
-      cacheGroups: bpkReactScriptsConfig.vendorsChunkRegex
-        ? {
-            defaultVendors: {
-              test: new RegExp(bpkReactScriptsConfig.vendorsChunkRegex)
-            },
-          }
-        : {},
-      }
+    ? isEnvDevelopment
+      ? {
+          ...chunksAndGroups,
+        }
+      : {
+          ...chunksAndGroups,
+          name: false,
```

Comment:
what is the chunk name looks like for both the name: false and without name case?

Reply:
when name is false:
<img width="555" alt="image" src="https://user-images.githubusercontent.com/86700834/167608934-e7f430cb-a0bf-47f9-bad6-3f5031a8838e.png">

when without name:
<img width="555" alt="image" src="https://user-images.githubusercontent.com/86700834/167608772-4d515f72-4bb0-444f-936b-7e6b6e317026.png">
So the chunk name is the same when the name: false and without name.